index.js

```diff
@@ -17,6 +19,16 @@ var TitleBar = function(options) {
 	events.EventEmitter.call(this);
 	this._options = options || {};
 
+	if (this._options.os == 'mac' && this._options.os == 'win'){
+		if (process.platform == 'darwin') this._options.os = 'mac';
+		else if (process.platform == 'win32') this._options.os = 'win';
+		else {
+			console.error('No supported os style type was given. Using OS X style as default.')
+			this._options.os = 'mac';
+		}
+	}
```

Comment:
Oh shoot, I meant to check if the option was neither 'mac' nor 'win', but I must not have tested that. I can redo that part if you need.